aspect/intellij_info_impl.bzl

```diff
@@ -327,12 +327,18 @@ def _collect_generated_proto_go_sources(target):
     if hasattr(target, "proto_go_api_info"):
         go_proto_info = target.proto_go_api_info
         files = getattr(go_proto_info, "files_to_build", depset()).to_list()
-    elif hasattr(target[OutputGroupInfo], "go_generated_srcs"):
-        files = target[OutputGroupInfo].go_generated_srcs.to_list()
     else:
         return None
     return [f for f in files if f.basename.endswith(".pb.go")]
 
+def _collect_generated_go_sources(target):
+    """Returns a depset of go source files generated by this target."""
+    if hasattr(target[OutputGroupInfo], "go_generated_srcs"):
+        files = target[OutputGroupInfo].go_generated_srcs.to_list()
+    else:
+        return None
+    return [f for f in files if f.basename.endswith(".go")]
+
```

Comment:
We can probably just rename the original function to this more generic name, and just have a separate file extension filtering and return for each if branch.

Reply:
Resolved with the new commit.